server/__init__.py

```diff
@@ -121,7 +121,8 @@ def report_dirties():
                 # allowed to see them.
                 if game.visibility == VisibilityState.FRIENDS:
                     # To see this game, you must have an authenticated connection and be a friend of the host, or the host.
-                    validation_func = lambda lobby_conn: lobby_conn.player.id in game.host.friends or lobby_conn.player == game.host
+                    validation_func = lambda \
+                        lobby_conn: lobby_conn.player.id in game.host.friends or lobby_conn.player == game.host
```

Comment:
Imho, you should leave first lambda param on same line as lambda. How do you read code? I read first input params then logic

Reply:
That's a good suggestion. I think this was a "line too long" formatting change, so I can change this to be on one line.

Reply:
How about just defining the function with a `def`? I don't see the point of using a lambda function here anyways.